frontend/test/metabase/scenarios/dashboard/dashboard-drill.cy.spec.js

```diff
@@ -257,89 +257,74 @@ describe("scenarios > dashboard > dashboard drill", () => {
+    const filter = {
+      id: "18024e69",
+      name: "Category",
+      slug: "category",
+      type: "category",
+    };
+
+    beforeEach(() => {
+      // Set "Rating" Field type to: "Category"
+      cy.request("PUT", `/api/field/${REVIEWS.RATING}`, {
+        semantic_type: "type/Category",
+      });
+
+      cy.createQuestionAndDashboard({ questionDetails }).then(
+        ({ body: { id, card_id, dashboard_id } }) => {
+          // Add filter to the dashboard
+          cy.request("PUT", `/api/dashboard/${dashboard_id}`, {
+            parameters: [filter],
+          });
+
+          // Connect filter to the dashboard card
+          cy.request("PUT", `/api/dashboard/${dashboard_id}/cards`, {
             cards: [
               {
-                id: dashCardId,
-                card_id: questionId,
+                id,
+                card_id,
                 row: 0,
                 col: 0,
                 sizeX: 8,
                 sizeY: 6,
                 parameter_mappings: [
                   {
-                    parameter_id: "18024e69",
-                    card_id: questionId,
+                    parameter_id: filter.id,
+                    card_id,
                     target: ["dimension", ["field", REVIEWS.RATING, null]],
                   },
                 ],
               },
             ],
           });
-        });
-
-        // NOTE: The actual "Assertion" phase begins here
-        cy.log("Reported failing on Metabase 1.34.3 and 0.36.2");
-
-        cy.log("The first case");
-        // set filter values (ratings 5 and 4) directly through the URL
-        cy.visit(`/dashboard/${dashboardId}?category=5&category=4`);
 
-        // drill-through
-        cy.findByText("xavier").click();
-        cy.findByText("=").click();
-
-        cy.findByText("Reviewer is xavier");
-        cy.findByText("Rating is equal to 2 selections");
-        cy.contains("Reprehenderit non error"); // xavier's review
+          // set filter values (ratings 5 and 4) directly through the URL
+          cy.visit(`/dashboard/${dashboard_id}?category=5&category=4`);
+          cy.findByText("2 selections");
+        },
+      );
+    });
 
-        cy.log("The second case");
-        // go back to the dashboard
-        cy.visit(`/dashboard/${dashboardId}?category=5&category=4`);
-        cy.findByText("2 selections");
+    it("when clicking on the field value (metabase#13062-1)", () => {
+      cy.findByText("xavier").click();
+      cy.findByText("=").click();
 
-        cy.findByText("13062Q").click(); // the card title
-        cy.findByText("Rating is equal to 2 selections");
-        cy.contains("Ad perspiciatis quis et consectetur."); // 5 star review
-      });
+      cy.findByText("Reviewer is xavier");
+      cy.findByText("Rating is equal to 2 selections");
+      cy.contains("Reprehenderit non error"); // xavier's review
+    });
+    it("when clicking on the card title (metabase#13062-2)", () => {
```

Comment:
I'd just add a line space between each it…

Reply:
Absolutely agreed. I have a plugin installed that sometimes visually tricks me that there is a space where there isn't.